pkg/execution/realtime/realtime.go

```diff
@@ -206,21 +233,39 @@ type Message struct {
 	Kind MessageKind `json:"kind"`
 	// Data represents the data in the message.
 	Data json.RawMessage `json:"data"`
+	// CreatedAt is the time that this message was created.
+	CreatedAt time.Time `json:"created_at"`
 
-	// FnID is the function ID that this message is related to.
-	FnID uuid.UUID `json:"fn_id,omitempty,omitzero"`
-	// FnSlug is the function slug that this message is related to.
-	FnSlug string `json:"fn_slug,omitempty,omitzero"`
-	// RunID is the run ID that this message is related to.
-	RunID ulid.ULID `json:"run_id,omitempty,omitzero"`
```

Comment:
Can we keep `RunID` in the `Message`?

`Channel` is the run ID atm but that could change, and keeping the `RunID` here means we also don't need to do `ulid.MustParse()` checks on `Channel` and it can remain _just a string_ which feels better.